core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -305,6 +313,34 @@ class ExecutionJsonSupport extends JsonSupport {
   implicit val ActiveSubmissionFormat = jsonFormat3(ActiveSubmission)
 
   implicit val WorkflowQueueStatusResponseFormat = jsonFormat3(WorkflowQueueStatusResponse)
+
+  implicit object WorkflowQueueStatusByUserResponseFormat extends RootJsonFormat[WorkflowQueueStatusByUserResponse] {
```

Comment:
Originally this was a one-liner:
```
implicit val WorkflowQueueStatusByUserResponseFormat = jsonFormat4(WorkflowQueueStatusByUserResponse)
```
...which worked, but I needed a custom formatter to conform to the JSON schema in [GAWB-2021](https://broadinstitute.atlassian.net/browse/GAWB-2021). I think this JSON format is more clear anyway.